Source/WebCore/PAL/Configurations/PAL.xcconfig

```diff
@@ -67,5 +67,55 @@ DAV1D_LIBTOOLFLAGS_YES = $(BUILT_PRODUCTS_DIR)/libdav1d.a;
+SWIFT_OBJC_INTEROP_MODE_MACOS_SINCE_1400 = objcxx;
+SWIFT_OBJC_INTEROP_MODE_MACCATALYST_SINCE_14 = objcxx;
+SWIFT_OBJC_INTEROP_MODE_IOS_SINCE_17 = objcxx;
+SWIFT_OBJC_INTEROP_MODE_WATCHOS_SINCE_10 = objcxx;
+SWIFT_OBJC_INTEROP_MODE_TVOS_SINCE_17 = objcxx;
+
+OTHER_SWIFT_FLAGS = $(OTHER_SWIFT_FLAGS_$(WK_PLATFORM_NAME));
+OTHER_SWIFT_FLAGS_macosx = $(OTHER_SWIFT_FLAGS$(WK_MACOS_1400));
+OTHER_SWIFT_FLAGS_maccatalyst = $(OTHER_SWIFT_FLAGS$(WK_MACCATALYST_14));
+OTHER_SWIFT_FLAGS_iphoneos = $(OTHER_SWIFT_FLAGS$(WK_IOS_17));
+OTHER_SWIFT_FLAGS_iphonesimulator = $(OTHER_SWIFT_FLAGS$(WK_IOS_17));
+OTHER_SWIFT_FLAGS_appletvos = $(OTHER_SWIFT_FLAGS$(WK_TVOS_17));
+OTHER_SWIFT_FLAGS_appletvsimulator = $(OTHER_SWIFT_FLAGS$(WK_TVOS_17));
+OTHER_SWIFT_FLAGS_watchos = $(OTHER_SWIFT_FLAGS$(WK_WATCHOS_10));
+OTHER_SWIFT_FLAGS_watchsimulator = $(OTHER_SWIFT_FLAGS$(WK_WATCHOS_10));
+OTHER_SWIFT_FLAGS_visionos = -Xcc -std=c++20 -Xcc -DSWIFT_ONLY -I $(SRCROOT)/pal;
+OTHER_SWIFT_FLAGS_visionsimulator = -Xcc -std=c++20 -Xcc -DSWIFT_ONLY -I $(SRCROOT)/pal;
+OTHER_SWIFT_FLAGS_MACOS_SINCE_1400 = -Xcc -std=c++20 -Xcc -DSWIFT_ONLY -I $(SRCROOT)/pal;
+OTHER_SWIFT_FLAGS_MACCATALYST_SINCE_14 = -Xcc -std=c++20 -Xcc -DSWIFT_ONLY -I $(SRCROOT)/pal;
+OTHER_SWIFT_FLAGS_IOS_SINCE_17 = -Xcc -std=c++20 -Xcc -DSWIFT_ONLY -I $(SRCROOT)/pal;
+OTHER_SWIFT_FLAGS_WATCHOS_SINCE_10 = -Xcc -std=c++20 -Xcc -DSWIFT_ONLY -I $(SRCROOT)/pal;
+OTHER_SWIFT_FLAGS_TVOS_SINCE_17 = -Xcc -std=c++20 -Xcc -DSWIFT_ONLY -I $(SRCROOT)/pal;
+
+SWIFT_ACTIVE_COMPILATION_CONDITIONS = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS_$(WK_PLATFORM_NAME));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_macosx = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_MACOS_1400));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_maccatalyst = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_MACCATALYST_14));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_iphoneos = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_IOS_17));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_iphonesimulator = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_IOS_17));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_appletvos = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_TVOS_17));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_appletvsimulator  = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_TVOS_17));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_watchos = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_WATCHOS_10));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_watchsimulator = $(SWIFT_ACTIVE_COMPILATION_CONDITIONS$(WK_WATCHOS_10));
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_visionos = SWIFTCPP_INTEROP;
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_visionsimulator = SWIFTCPP_INTEROP;
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_MACOS_SINCE_1400 = SWIFTCPP_INTEROP;
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_MACCATALYST_SINCE_14 = SWIFTCPP_INTEROP;
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_IOS_SINCE_17 = SWIFTCPP_INTEROP;
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_WATCHOS_SINCE_10 = SWIFTCPP_INTEROP;
+SWIFT_ACTIVE_COMPILATION_CONDITIONS_TVOS_SINCE_17 = SWIFTCPP_INTEROP;
```

Comment:
What is this for? I don't see a `SWIFTCPP_INTEROP` condition checked anywhere.

Reply:
see `CryptoKitShim.swift`

Reply:
My bad, large diff = hidden by default. Thanks!